dpcpp/base/executor.dp.cpp

```diff
@@ -234,10 +235,24 @@ void delete_queue(sycl::queue* queue)
 }
 
 
+::cl::sycl::property_list get_property_list(dpcpp_queue_property property)
+{
+    if (property == dpcpp_queue_property::in_order) {
+        return {sycl::property::queue::in_order{}};
+    } else if (property == dpcpp_queue_property::enable_profiling) {
+        GKO_NOT_SUPPORTED(property);
+        return {sycl::property::queue::enable_profiling{}};
```

Comment:
why return something if you fail in the line before?

Reply:
this is to avoid the warning about the condition return nothing matching the return type

Reply:
the compiler should already figure out that this code is unreachable: https://godbolt.org/z/TMnbKPbY4

Reply:
I see. thanks for the example.